ocs_ci/ocs/ui/page_objects/storage_clients.py

```diff
@@ -12,15 +14,34 @@ class StorageClients(BaseUI):
 
     def __init__(self):
         super().__init__()
+        self.ocs_version = version.get_semantic_ocs_version_from_config()
 
-    def generate_client_onboarding_ticket(self):
+    def generate_client_onboarding_ticket_ui(self, storage_quota=None):
         """
         Generate a client onboarding ticket
 
         Returns:
             str: onboarding_key
         """
+        logger.info("Generating onboarding token from ui")
         self.do_click(self.storage_clients_loc["generate_client_onboarding_ticket"])
+        ValidationUI().verify_storage_clients_page()
```

Comment:
I think it would be better not to run verification of storage clients page by default each time we generate a token in the UI.